AdobeAnaltyicsAPI.gs

```diff
@@ -128,6 +158,67 @@ var AdobeAnalyticsAPIController = (function() {
 
   };
 
+
+  /**
+   * AdobeAnalyticsAPIController.prototype.getJwtToken()
+   * get JSON Web Token
+   *
+   * @return {String} JSON Web Token
+   **/
+  AdobeAnalyticsAPIController.prototype.getJwtToken = function () {
+  
+    this.Logger("", {
+      "title": "getJwtToekn"
```

Comment:
I think maybe typo.  it's "getJwtToken".

Reply:
Thank you for pointing it out. I'll fix the typo.